### ISSUE TITLE
Bug: OmniSharp.Roslyn.CSharp.Workers.Diagnostics.AnalyzerWorkQueue fails to load

### ISSUE THREAD

**Issue by Siccity** [OmniSharp/omnisharp-roslyn#1983]
I'm loading a Unity project the same way I've been doing for the past 3 years, but today it stopped working. Instead, it shows me this message over and over in the console.
```
[warn]: OmniSharp.Roslyn.CSharp.Workers.Diagnostics.AnalyzerWorkQueue
        Timeout before work got ready for foreground analysis queue. This is assertion to prevent complete api hang in case of error.
```
I have no idea how to fix this.

**Comment by jannickj** [comment]
I am experience the exact same :(

**Comment by chickenSandwich** [comment]
Same:
```
OmniSharp.Roslyn.CSharp.Workers.Diagnostics.AnalyzerWorkQueue
        Timeout before work got ready for foreground analysis queue. This is assertion to prevent complete api hang in case of error.
```

**Comment by Siccity** [comment]
I noticed they mentioned this issue under 'Known Issues' in the changelog
![image](https://user-images.githubusercontent.com/6402525/95958838-01ed8980-0e02-11eb-89b5-43f7838082bb.png)
Changing that setting didn't fix it for me, but reverting to an older version of the extension did.

**Comment by filipw** [comment]
This is unrelated. The fix is in #1984
